packages/embark-console/src/test/console.js

```diff
@@ -1,25 +1,38 @@
 /*globals describe, it*/
-let Console = require('../lib/modules/console/');
-let Plugins = require('../lib/core/plugins.js');
-let IPC = require('../lib/core/ipc.js');
-let Logger = require('embark-logger');
-let assert = require('assert');
-let version = require('../../package.json').version;
+import Console from '../lib';
+import Logger from 'embark-logger';
+import assert from 'assert';
+import { version }  from '../../package.json';
+import * as i18n from "i18n";
+
+i18n.configure({
```

Comment:
In the test it's important to configure `i18n` else there's an exception re: `logWarnFn`.

Reply:
Shit, this is where I was getting stuck. We need an `embark-tests` package that exposes test helpers and does this sort of groundwork. Good catch.